packages/terra-consumer-nav/src/Nav.jsx

```diff
@@ -5,10 +5,11 @@ import Button from 'terra-button';
 import IconClose from 'terra-icon/lib/icon/IconClose';
 import NavItems from './components/nav-items/NavItems';
 import NavLogo from './components/nav-logo/NavLogo';
-import NavProfile from './components/nav-profile/NavProfile';
+import navElementShape from './NavPropShapes';
```

Comment:
Whats the difference between navElementShape and navItemShape? The naming is pretty hard to tell any significant difference. It appears navItems can have badges and active status?